modules/upi-installation-considerations.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting-installations.adoc
+
+[id="upi-installation-considerations_{context}"]
+= User-provisioned infrastructure installation considerations
+
+The default installation method uses installer-provisioned infrastructure. With installer-provisioned infrastructure clusters, {product-title} manages all aspects of the cluster, including the operating system itself. If possible, use this feature to avoid having to provision and maintain the cluster infrastructure.
+
+You can alternatively install {product-title} {product-version} on infrastructure that you provide. If you use this installation method, follow user-provisioned infrastructure installation recommendations carefully.
```

Comment:
We should try and expand on what specific things users need to consider. 

https://access.redhat.com/articles/4207611 is a good starting point for the types of thing users should consider / be ready to investigate when using UPI.

Reply:
This is very helpful. I have now expanded this module extensively based on the details included in that article. Thanks!